app/static/ie-status.json

```diff
@@ -4228,6 +4228,33 @@
     "uservoiceid": null
   },
   {
+    "name": "Backdrop filter",
+    "category": "CSS",
+    "link": "http://dev.w3.org/fxtf/filters-2/#BackdropFilterProperty",
+    "summary": "Applies a CSS filter effect to the content behind an element. This is not the element's background but the content that would be drawn behind it.",
+    "standardStatus": "Working draft or equivalent",
+    "ieStatus": {
+      "text":"Not currently planned",
+      "iePrefixed": "",
+      "ieUnprefixed": ""
+    },
+    "impl_status_chrome": "In Development",
+    "ff_views": {
+      "text": "No public signals",
+      "value": 5
+    },
+    "safari_views": {
+      "text": "Preview release",
```

Comment:
We don't have preview release status for other browsers. So this should be "In Development" for Safari.